### PR TITLE
Frequently used reacji bar integration

### PR DESCRIPTION
**Pull request by buoyad**
r? @keybase/react-hackers

### CODE REVIEW

File: shared/chat/conversation/messages/react-button/emoji-row/data.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// @flow
+import {memoize} from '../../../../../util/memoize'
+import {isMobile} from '../../../../../constants/platform'
+
+const getFrequently = memoize<void, void, void, void, {get: number => Array<string>}>(
+  () => require('emoji-mart').frequently
```
Comment (zanderz) [inline shared/chat/conversation/messages/react-button/emoji-row/data.js]: This means we rely on some facility of emoji-mart to say which ones are frequent>

**Comment by buoyad** [comment]
@zanderz yep that is the designs, it works like that for me, are you up to date? [This change](https://github.com/keybase/client/pull/15622/commits/f22f6ad1db3f5a6bfc25663044757664dc123530) hides it when there are existing reacjis

File: shared/chat/conversation/messages/react-button/emoji-row/data.js (hunk 1 above)
Comment (buoyad) [reply shared/chat/conversation/messages/react-button/emoji-row/data.js]: Yeah it does. It uses `LocalStorage` which we can do when we transition away from emoji-mart

**Comment by zanderz** [comment]
I tried `danny/DESKTOP-8657-integrate-2` within the last hour. It is NBD but it just looked like it is supposed to pop up in a different place depending on where you hover but it is not worth worrying about I think. SHIP IT!

**Comment by buoyad** [comment]
Cool thanks @zanderz, I still need review on #15592 though 😄

File: shared/chat/conversation/messages/react-button/emoji-row/index.js [hunk 2]
```diff
@@ -74,6 +96,7 @@ const styles = Styles.styleSheetCreate({
       borderRadius: 4,
       borderStyle: 'solid',
       borderWidth: 1,
+      height: 24,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-row/index.js]: globalMargins.medium

File: shared/chat/conversation/messages/react-button/emoji-row/index.js [hunk 3]
```diff
@@ -89,6 +112,13 @@ const styles = Styles.styleSheetCreate({
     paddingRight: Styles.globalMargins.xsmall,
     paddingTop: Styles.globalMargins.xtiny,
   },
+  pickerContainer: Styles.platformStyles({
+    isElectron: {
+      borderRadius: 4,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-row/index.js]: Styles.borderRadius

**Pull request merged**